src/test/java/pkgGame/SudokuTest.java

```diff
@@ -89,48 +92,169 @@ public void Cell_Test() throws Exception {
 	@Test
 	public void Cell_getiRow_Test() throws Exception {
 		
-		// TODO: Yisi will do this...
-		// ...
+		// TODO: Aris will do this...
 		
-		Sudoku.Cell s = new Sudoku(9);
+		/*
+		Sudoku.Cell s = new Sudoku.Cell(6,7);
+		
+		assertTrue(s.getiRow() == 6);
+		*/
+		
+		try {
+			Class<?> c = Class.forName("pkgGame.Sudoku.Cell");
+			Constructor constructor = c.getConstructor(new Class[] { int.class, int.class });
+			constructor.setAccessible(true);
+			s = (Sudoku.Cell) constructor.newInstance(6, 7);
+
+			Method mgetiRow = c.getiRow();
+			
+			assertTrue(mgetiRow.invoke() == 6);
+		} catch (ClassNotFoundException e1) {
+			fail("ClassNotFoundException");
+		} catch (NoSuchMethodException e) {
+			fail("NoSuchMethodException");
+		} catch (SecurityException e) {
+
+			fail("SecurityException");
+		} catch (InstantiationException e) {
+			fail("InstantiationException");
+		} catch (IllegalAccessException e) {
+			fail("IllegalAccessException");
+		} catch (IllegalArgumentException e) {
+			fail("IllegalArgumentException");
+		} catch (InvocationTargetException e) {
+			fail("InvocationTargetException, Invalid size");
+		}
 		
-		//assertTrue(...);
-
 	}
 	
 	@Test
 	public void Cell_getiCol_Test() throws Exception {
 		
-		// TODO: Yisi will do this...
+		// TODO: Aris will do this...
 		// ...
+		/*
+		Sudoku.Cell s = new Sudoku.cell(2,3);
 		
-		Sudoku.Cell s = new Sudoku(9);
+		assertTrue(s.getiCol() == 3);
+		*/
+		try {
+			Class<?> c = Class.forName("pkgGame.Sudoku.Cell");
+			Constructor constructor = c.getConstructor(new Class[] { int.class, int.class });
+			constructor.setAccessible(true);
+			s = (Sudoku.Cell) constructor.newInstance(2, 3);
+
+			Method mgetiCol = c.getiCol();
+			
+			assertTrue(mgetiCol.invoke() == 3);
+		} catch (ClassNotFoundException e1) {
+			fail("ClassNotFoundException");
+		} catch (NoSuchMethodException e) {
+			fail("NoSuchMethodException");
+		} catch (SecurityException e) {
+
+			fail("SecurityException");
+		} catch (InstantiationException e) {
+			fail("InstantiationException");
+		} catch (IllegalAccessException e) {
+			fail("IllegalAccessException");
+		} catch (IllegalArgumentException e) {
+			fail("IllegalArgumentException");
+		} catch (InvocationTargetException e) {
+			fail("InvocationTargetException, Invalid size");
+		}

```

Comment:
Maybe these two unit tests could be abstract to one function since they're so similar, otherwise okay.